low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -63,90 +63,99 @@ def validate():
+
+            os.makedirs(service_executable_code_des, 0o777, exist_ok=True)
+
+            shutil.copy(service_executable_code_src,
+                        f'{service_executable_code_des}/dummy_service.py')
+            # Make service file executable.
+            cmd = f"chmod +x {service_executable_code_des}/dummy_service.py"
+            _, error, returncode = SimpleProcess(cmd).run()
+            if returncode !=0:
+                print("%s error occurred while executing cmd: %s" %(error, cmd))
+                print("failed to assign execute permission for dummy_service.py."\
+                        " dummy_service will fail.")
+
+            # Copy service file to /etc/systemd/system/ path.
+            shutil.copyfile(service_file_path_src,
+                            f'{service_file_path_des}/dummy_service.service')
+            cmd= "systemctl daemon-reload"
+            _, error, returncode = SimpleProcess(cmd).run()
+            if returncode !=0:
+                    print(f"failed to execute '{cmd}', systemctl will be unable"\
+                        f" to manage the dummy_service.service \nError: {error}")
+
+            self.dbus_service.enable(service_name)
+            self.dbus_service.start(service_name)
+
+            service_list = Conf.get(SSPL_CONFIG_INDEX, "SERVICEMONITOR>monitored_services")
+            service_list.append(service_name)
+            Conf.set(SSPL_CONFIG_INDEX, "SERVICEMONITOR>monitored_services",
+                service_list)
+
+            threshold_inactive_time_original = Conf.get(SSPL_CONFIG_INDEX,
+                                        "SERVICEMONITOR>threshold_inactive_time")
+            threshold_inactive_time_new = 30
+            Conf.set(SSPL_CONFIG_INDEX, "SERVICEMONITOR>threshold_inactive_time",
+                threshold_inactive_time_new)
+            Conf.save(SSPL_CONFIG_INDEX)
+
+            # TODO: Convert shell script to python
+            # from cortx.sspl.sspl_test.run_qa_test import RunQATest
+            # RunQATest(self.plan, self.avoid_rmq).run()
+            CMD = "%s/run_qa_test.sh %s %s" % (TEST_DIR, self.plan, self.avoid_rmq)
+            output, error, returncode = SimpleProcess(CMD).run(realtime_output=True)
+            # Restore the original path/file & service, then throw exception
+            # if execution is failed.
+            service_list.remove(service_name)
+            Conf.set(SSPL_CONFIG_INDEX,"SERVICEMONITOR>monitored_services",
+                service_list)
+            Conf.set(SSPL_CONFIG_INDEX, "SERVICEMONITOR>threshold_inactive_time",
+                threshold_inactive_time_original)
+            Conf.set(SSPL_CONFIG_INDEX,
+                    "SYSTEM_INFORMATION>global_config_copy_url", global_config_copy_url)
+            Conf.save(SSPL_CONFIG_INDEX)
+            shutil.copyfile(sspl_test_backup, sspl_test_file_path)
+
+            self.dbus_service.restart('sspl-ll.service')
```

Comment:
Please remove this restart and verify once again.

Reply:
Tried sanity after removing restart, sanity will run without any issue,
But observed one thing ie, For sanity test, after test completion  we restored changed config to original config, (in run_qa_test.sh script-restored_config).
So if we remove this restart then that restored config will not reflect in service, Service will still used test config.

Reply:
I have removed this in PR #347 
Please rebase your repo and check this scenario.